### ISSUE TITLE
Disable redirect for certain templates

### ISSUE THREAD

**Issue by zamson** [instantcommerce/shopify-headless-theme#23]
I'm trying to set up some custom xml feeds, is it possible to disable redirects for certain templates / paths?

**Comment by zamson** [comment]
Answered my own question, used the method in the gift card template applying:

`{% layout none %}`

**Comment by DundarKoray** [comment]
Yes, there should be any easy way to implement disabling redirects. 
For example, if I want to go to Login page or orders page.

**Comment by leeuwis** [comment]
`{% layout none %}` is indeed the easiest way. Allowing to add a list of pages/urls to disable redirects like @DundarKoray suggests seems like a nice addition. Feel free to open a PR and I'll review it!
